src/App.vue

```diff
@@ -558,6 +558,7 @@ export default {
       this.setLastDirectorChangeDate(business.lastDirectorChangeDate) // may be empty
       this.setWarnings(Array.isArray(business.warnings) ? business.warnings : [])
       this.setGoodStanding(business.goodStanding)
+      this.setHasCourtOrders(business.hasCourtOrders)
```

Comment:
Is this property always available in the business object? If not, would it make sense to fall back to False here?

Reply:
Great, then this is fine.